pgcli/main.py

```diff
@@ -215,8 +213,7 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self._completer_lock = threading.Lock()
         self.register_special_commands()
 
-        self.eventloop = create_eventloop()
-        self.cli = None
+        self.prompt_app = None
```

Comment:
Maybe this can be removed (or changed to a prompt_app_initialized boolean) as we're using a `get_app()` singleton now.

Reply:
I'm not sure about that one. Seems to me like one of those things that's not broken and no reason to fix.

Reply:
sorry for the confusion; prompt_app is a `prompt_toolkit.shortcuts.PromptSession`, which cannot be reached from the get_app() singleton.

Reply:
ah ok, good to know!